force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -102,16 +126,24 @@ public with sharing class ClassesObjectsMethods {
      */
     public static Integer countVowels(String str) {
         // Initialize the result Integer
-
-        /*
-        for () {
+        Integer vowelCount = 0;
+        str = str.toLowerCase();
+        
+        for (Integer i = 0; i < str.length(); i++) {
+            String ch = str.substring(i, i + 1);
+            
+        
+        if (ch == 'a' || ch == 'e' || ch == 'i' || ch == 'o' || ch == 'u') {
+            vowelCount++;
+        }
+    }
             // get individual characters from the string - hint use substring or split
             // check if the character is a vowel
             // if it is a vowel, increment the result Integer
-        }
-        */
+        
+        
 
-        return null; // Replace null with the variable you used to store the result
+        return vowelCount;// Replace null with the variable you used to store the result
```

Comment:
Can you solve this with a set rather than conditional logic? When you are checking for membership, this is often the easiest way to go and easier to read than lots of  ORs and ANDs